assets/javascript/app.js

```diff
@@ -7,12 +7,13 @@ $(document).ready(function() {
     var lat;
     var long;
     var brewer;
+    var filter = false
 
     // Function that grabs latitude and longitude of user from ip address
     function ipLookUp () {
         var options = {
             enableHighAccuracy: true,
-            timeout: 5000,
+            timeout: 60000,
```

Comment:
Updated the timeout to 1 minute because users that do not accept the geo request within 5 seconds auto default to NYC per our logic. If they accept after the 5 seconds then it doesn't update the map as the request had already timed out and we built the map using our fallback coordinates. 

This gives the user a little more time to accept, especially if they see the age verification form first and not the geo request. Could probably go longer, or be removed altogether but this is a start. Open to thoughts on changes here.